### ISSUE TITLE
loom file

### ISSUE THREAD

**Issue by iiiir** [bicycle1885/CellFishing.jl#6]
I wonder if you can kindly provide some test loom files?
for example Plass2018.dge.loom.

Thanks,
Shuoguo

**Comment by zhanglab2008** [comment]
Agreed. i also want to test a loom file. I used my loom file and it doesn't work. Can you provide one as an example?
